test/model/recent_senders_test.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/model/recent_senders.dart';
+import '../example_data.dart' as eg;
+
+void main() {
+  group('MessageIdTracker', () {
```

Comment:
nit: order tests to match the code under tests — so this group after the RecentSenders group, given that the MessageIdTracker class comes after the RecentSenders class